src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const thead = table.querySelector('thead');
+const trList = [...table.querySelectorAll('tr')];
```

Comment:
```suggestion
const trList = [...table.querySelectorAll('tbody tr')];
```
И тогда не надо будет делать slice